todo_app/data/trello_items.py

```diff
@@ -0,0 +1,114 @@
+import requests
+import os
+
+
+# board_id = os.getenv(TRELLO_BOARD_ID)
+# trello_token = os.getenv(TRELLO_TOKEN)
+# trello_key = os.getenv(TRELLO_KEY)
+
+class Trello:
+    def __init__(self):
+        self.board_id = os.getenv('TRELLO_BOARD_ID')
+        self.trello_token = os.getenv('TRELLO_TOKEN')
+        self.trello_key = os.getenv('TRELLO_KEY')
+        self.lists = self.get_lists()
```

Comment:
This is a really nice pattern, but you may notice that your app is fairly slow, which I suspect is because each time I make any request, it loads all the lists up fresh. One way you could work round that is to put the `get_items` etc. functions below into this class, so that you only load the lists once, and then share them. E.g. the get_items function would then look more like:
```python
    def get_items(self):
        """
        Fetches all items on Trello board via API

        Returns:
            list: The list of Trello items.
        """
        trello_get_cards_api = requests.get(
            f"https://api.trello.com/1/boards/{self.board_id}/cards/open?key={self.trello_key}&token={trello_details.trello_token}").json()
        items = self._select_fields(trello_get_cards_api, self.lists)
        return items
```

Having said that, I tested and actually there's a bigger culprit below - see the comment on `Item`!

Reply:
I have moved the functions into the Trello class